test-workspaces/sfdx-multi-workspace/sfdx-workspace-1/.sfdx/typings/lwc/apex.d.ts

```diff
@@ -0,0 +1,31 @@
+declare module "@salesforce/apex" {
```

Comment:
Probably should delete this entire .sfdx/typings/ directory to ensure it gets created each time during tests